.github/actions/create-prod-env-if-not-exist/action.yaml

```diff
@@ -30,6 +34,7 @@ runs:
       
     - name: Create production environment if not exists
       shell: bash
+      working-directory: ${{ inputs.working-directory }}
       run: |
         # Disable exit on error
         set +e
```

Comment:
We should cd to the working directory and run the rest of the script relative to that